features/support/ui/pages/money_manager.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../page'
+
+module UI::Pages
+  class MoneyManager < UI::Page
+    set_url '/en/tools/money-manager/'
+
+    element :received_first_payment_true,
+            '#questionnaire_received_first_payment_true'
```

Comment:
To be honest, I'm not sure what benefits that would give compared to a unique id, in terms of decoupling. I can see the value if you'd otherwise have to select by CSS classes that aren't unique, or that are cosmetic and could change, but when there is already an ID and it's unique, that seems like the best option.